craft_application/application.py

```diff
@@ -486,6 +486,7 @@ def run(  # noqa: PLR0912,PLR0915  (too many branches, too many statements)
     ) -> int:
         """Bootstrap and run the application."""
         self._setup_logging()
+        self._enable_craft_parts_features()
         self._register_default_plugins()
```

Comment:
this should ideally be "initialize_craft_parts", the ordering of the methods annoys me a bit along the code, but that is a different story

Reply:
that's a good idea, added `_initialize_craft_parts()` with `_enable_craft_parts_features()` and `_register_default_plugins()` inside